Cineme.h

```diff
@@ -0,0 +1,116 @@
+
+#include "Movie.h"
+
+
+class Cinema
+{
+private:
+    vector<Movie> movies;
+    string name;
+    
+    int seats;
+
+public:
```

Comment:
The class members should be wrapped by 4 spaces(or 1 tab).

Reply:
Changed